README.md

```diff
@@ -10,4 +10,8 @@ What I have learned about clean code in this subject
 - Don't repeat yourself
 
 
```

Comment:
Use #blah for largest font, then ##blah for next down and so on:

# CP5632_Programming02

## Practicals carried out for Master's course CP5632, Programming 2, at JCU Cairns.

### About Me